src/maintenance/remove_outdated_keys.py

```diff
@@ -0,0 +1,73 @@
+from collections import namedtuple
+from smtplib import SMTP
+
+import delphi.operations.secrets as secrets
+import mysql.connector
+from delphi.epidata.server._config import API_KEY_REGISTRATION_FORM_LINK_LOCAL
+
+API_USER_RECORD = namedtuple("APIUser", ("api_key", "email", "date_diff"))
```

Comment:
change the naming style because this is more like a class than a const
```suggestion
APIUserRecord = namedtuple("APIUserRecord", ("api_key", "email", "date_diff"))
```